bin/marathon

```diff
@@ -121,6 +121,23 @@ def subcmd_kill_tasks(cmd_opts)
   print_tasks(tasks)
 end
 
+def subcmd_rollback(cmd_opts)
+  app = Marathon::App.get(cmd_opts[:id])
+  # Get current versions
+  versions = app.versions
+  # Retrieve N-1 version if none given
+  target = cmd_opts[:version] ? cmd_opts[:version] : versions[1]
```

Comment:
You may have a typo there, since the option name in the Trollop.options block is `version_id` instead of `version`